tests/python/kaolin/ops/conversions/test_voxelgrid.py

```diff
@@ -1074,50 +1074,61 @@ def _flip(self, voxelgrid, expected_vertices, dims):
         else:
             new_voxelgrid = voxelgrid.flip(dims)
 
-        new_vertices, _ = vg.voxelgrids_to_trianglemeshes(new_voxelgrid.unsqueeze(0))
-
-        return new_vertices, new_expected_vertices, new_voxelgrid
+        return new_expected_vertices, new_voxelgrid
     
     def _rotate(self, voxelgrid, expected_vertices, dims):
 
         new_voxelgrid = voxelgrid.permute(dims)
-        new_vertices, _ = vg.voxelgrids_to_trianglemeshes(new_voxelgrid.unsqueeze(0))
 
         new_expected_vertices = expected_vertices.detach().clone()
 
         if dims == (0, 2, 1) or dims == [0, 2, 1]:
-            new_expected_vertices[:, 0], new_expected_vertices[:, 1] = expected_vertices[:, 1], expected_vertices[:, 0]
-        elif dims == (1, 0, 2) or dims == [1, 0, 2]:
             new_expected_vertices[:, 1], new_expected_vertices[:, 2] = expected_vertices[:, 2], expected_vertices[:, 1]
+        elif dims == (1, 0, 2) or dims == [1, 0, 2]:
+            new_expected_vertices[:, 0], new_expected_vertices[:, 1] = expected_vertices[:, 1], expected_vertices[:, 0]
+        elif dims == (1, 2, 0) or dims == [1, 2, 0]:
+            new_expected_vertices[:, 0] = expected_vertices[:, 1]
+            new_expected_vertices[:, 1] = expected_vertices[:, 2]
+            new_expected_vertices[:, 2] = expected_vertices[:, 0]
+            # pass
         elif dims == (2, 1, 0) or dims == [2, 1, 0]:
             new_expected_vertices[:, 0], new_expected_vertices[:, 2] = expected_vertices[:, 2], expected_vertices[:, 0]
-        
-        return new_vertices, new_expected_vertices, new_voxelgrid
+        elif dims == (2, 0, 1) or dims == [2, 0, 1]:
+            new_expected_vertices[:, 0] = expected_vertices[:, 2]
+            new_expected_vertices[:, 1] = expected_vertices[:, 0]
+            new_expected_vertices[:, 2] = expected_vertices[:, 1]
+
+        return new_expected_vertices, new_voxelgrid
 
     def _all_variations_test(self, voxelgrid, expected_vertices):
 
-        all_rotations = [[0, 1, 2], [1, 0, 2], [2, 1, 0]]
+        all_rotations = [[0, 1, 2], [0, 2, 1], [1, 0, 2], [1, 2, 0], [2, 1, 0], [2, 0, 1]]
         all_flips = [[], [0], [1], [2], [0, 1], [0, 2], [1, 2], [0, 1, 2]]
 
         for rotate in all_rotations:
-            vertices, expected_vertices, voxelgrid = self._rotate(voxelgrid, expected_vertices, rotate)
+            new_expected_vertices, new_voxelgrid = self._rotate(voxelgrid, expected_vertices, rotate)
             for flip in all_flips:
-                new_vertices, new_expected_vertices, _ = self._flip(voxelgrid, expected_vertices, flip)
-                new_vertices = torch.sort(new_vertices[0], dim=0)[0]
-                new_expected_vertices = torch.sort(new_expected_vertices, dim=0)[0]
+                curr_expected_vertices, curr_voxelgrid = self._flip(new_voxelgrid, new_expected_vertices, flip)
+                curr_vertices, _ = vg.voxelgrids_to_trianglemeshes(curr_voxelgrid.unsqueeze(0))
```

Comment:
Good to have the `voxelgrids_to_trianglemeshes` here instead of _rotate / _flip.